dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A database browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite. \
+It is for users and developers wanting to create databases, search, and edit data. \
+It uses a familiar spreadsheet-like interface, and you don't need to learn complicated\ SQL commands.\
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="COPYRIGHT 2012-2014 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+SOURCE_FILENAME="sqlitebrowser-$portVersion.tar.gz"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+    sqlitebrowser = $portVersion
+    libs:antlr-2.7.7
+    libs:qcustomplot-source
+    libs:qhexedit
+    libs:qscintilla
+    "
+REQUIRES="
+        haiku >= $haikuVersion
+        "
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsqlite3
+	devel:libqt5
+         devel:libphonon
+	devel:libQt3Support
+	devel:libQtCLucene
+	devel:libQtCore
+	devel:libQtDBus
+	devel:libQtDeclarative
+	devel:libQtDesigner
+	devel:libQtDesignerComponents
+	devel:libQtGui
+	devel:libQtHelp
+	devel:libQtMultimedia
+	devel:libQtNetwork
+	devel:libQtScript
+	devel:libQtScriptTools
+	devel:libQtSql
+	devel:libQtSvg
+	devel:libQtTest
+	devel:libQtWebKit
+	devel:libQtXml
+	devel:libQtXmlPatterns
+	devel:libQtUiTools
+         	"
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:assistant
+	cmd:designer
+	cmd:lconvert
+	cmd:linguist
+	cmd:lrelease
+	cmd:lupdate
+	cmd:moc
+	cmd:pixeltool
+	cmd:qcollectiongenerator
+	cmd:qdbuscpp2xml
+	cmd:qdbusviewer
+	cmd:qdbusxml2cpp
+	cmd:qdoc3
+	cmd:qhelpconverter
+	cmd:qhelpgenerator
+	cmd:qmake
+	cmd:qmlplugindump
+	cmd:qmlviewer
+	cmd:qt3to4
+	cmd:qttracereplay
+	cmd:rcc
+	cmd:uic
+	cmd:uic3
+	cmd:xmlpatterns
+	cmd:xmlpatternsvalidator
```

Comment:
That said, there are cases where you could:
1) find that you need a given command, let's call it "foo", so you add cmd:foo to BUILD_PREREQUIRES and start the build again.
2) Everything builds fine, but you did not notice that it also needs another command, let's call it bar, so you did not add cmd:bar to BUILD_PREREQUIRES.

This will happen if cmd:foo and cmd:bar are provided by the same package.
But if later that package gets split into sub-packages, with one providing cmd:foo and another providing cmd:bar, then your recipe won't work anymore, unless you add cmd:bar to BUILD_PREREQUIRES.

That almost never happens, so it's generally fine to add cmd: lines one by one, until the build succeeds.

The perfect approach is to do as @korli said, and to look at the output of the configure script when you start building. You will notice all the "Checking for" lines.
I don't know if this is recommended or not, but I tend to add in BUILD_PREREQUIRES the cmd items that the configure script tries to detect.
I know @korli does this for the most important items, and that's why I also do that ;-)
(I learned a lot with @korli (and still have a lot to learn, btw ;-)